apollo-portal/src/main/resources/static/scripts/controller/IndexController.js

```diff
@@ -119,6 +127,23 @@ function IndexController($scope, $window, $translate, toastr, AppUtil, AppServic
             })
     }
 
+    function getPublicProjects() {//TODO
```

Comment:
I noticed we have a lot of methods/variabled named with `PublicProjects`, however, I think they are actually `PublicNamespaces`? Shall we rename them to better reflect the actual meaning?